dict/prh-idiomatic-usage.yml

```
@@ -878,6 +887,43 @@ rules:
         to: レイヤー
       - from: Flashプレイヤー
         to: Flashプレイヤー
+  - expected: わか$2
+    pattern:
+      - /((?<![自区])分か(?!れる|れて|れた)|判|解)([らりるれっ])/
```

```suggestion
      - /((?<![自区差])分か(?!れる|れて|れた)|判|解)([らりるれっ])/
```
hanicaやkobanでは差分って言葉をよく使う気がしたので、念のため対象外にしても良さそうかなと思いました。